js/tokens.js

```diff
@@ -0,0 +1,496 @@
+            'status': status,
+            'symbol': symbol,
+            'type': 'limit',
+            'side': side,
+            'price': price,
+            'cost': cost,
+            'amount': amount,
+            'filled': filled,
+            'remaining': remaining,
+            'trades': trades,
+            'fee': fee,
+            'info': order,
+        };
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let request = {
+            'tradingPair': market['id'],
+            'amount': this.amountToPrecision (symbol, amount),
+            'side': side,
+            'price': this.priceToPrecision (symbol, price),
+        };
+        let response = await this.privatePostPrivateOrdersAddLimit (this.extend (request, params));
+        // let order = this.parseOrder (response, market);
+        let timestamp = (response['timestamp'] * 1000).toString ();
+        return {
+            'info': response,
+            'datetime': this.iso8601 (timestamp),
+            'timestamp': timestamp,
+            'id': this.safeString (response, 'orderId'),
+        };
+    }
+
+    async cancelOrder (id) {
+        return await this.privatePostPrivateOrdersCancelId ({ 'id': id });
+    }
+
+    async fetchOrderStatus (id) {
+        let order = await this.privateGetPrivateOrdersGetId ({ 'id': id });
+        return this.parseOrderStatus (this.safeString (order, 'orderStatus'));
+    }
+
+    async fetchOrder (id) {
+        await this.loadMarkets ();
+        let order = await this.privateGetPrivateOrdersGetId ({ 'id': id });
+        let parsed = this.parseOrder (order);
+        return parsed;
+    }
+
+    // async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {}
+
+    // async fetchTransactions (code = undefined, since = undefined, limit = undefined, params = {}) {}
+
+    // async fetchWithdrawals (code = undefined, since = undefined, limit = undefined, params = {}) {}
+
+    // async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {}
+
+    // async fetchDepositAddress (code, params = {}) {}
+
+    // async withdraw (code, amount, address, tag = undefined, params = {}) {}
+
+    // 100 API Key is missing
+    // 101 Nonce is missing
+    // 102 Signature is missing
+    // 110 Nonce has to be integer
+    // 111 Provided nonce is less or equal to the last nonce
+    // 120 Invalid API key
+    // 121 Signature is invalid
+    // 130 Invalid trading pair
+    // 131 Invalid order id
+    // 140 Only opened orders can be canceled
+    // 150 Parameter {parameter} is invalid with error: {error}
+    // 160 Invalid currency code
+    // 429 API rate limit exceeded
+    handleErrors (httpCode, reason, url, method, headers, body, response) {
+        switch (response.errorCode) {
+        case 100:
```

Comment:
Need to move these error codes to `exceptions` in `describe()`, see how that is done here:

- https://github.com/ccxt/ccxt/blob/master/js/liqui.js#L76
- https://github.com/ccxt/ccxt/blob/master/js/liqui.js#L801

- https://github.com/ccxt/ccxt/blob/master/js/binance.js#L277
- https://github.com/ccxt/ccxt/blob/master/js/binance.js#L1191